### DISCUSSION TITLE
posts/tutorial5/

### DISCUSSION THREAD

**Discussion by giscus** [7w1/7w1.github.io#3] [Announcements]
# posts/tutorial5/

Now that we’ve built some survey ships, let’s send them off to survey.

https://7w1.github.io/posts/tutorial5/

**Comment by Khaligufzel** [comment]
Dang it! No more tutorials!? :<

Well, have to figure out the rest then!
Thank you for your work!

**Comment by miosad** [comment]
great stuff 👍👍👍

**Comment by stupidlogins** [comment]
Where part 6

**Comment by kittyk4y** [comment]
Very sad there isn't more tutorials :( This was super helpful!

**Comment by Obiben27** [comment]
Thank you for how far you got me here. glad it still mostly is the same

**Comment by yfyldfylfhkcvkyd** [comment]
slightly outdated but still useful :3 with out it i couldn't figure out by myself how to use these goddamn shipyards

**Comment by QuestionableDocumentation** [comment]
Shame this is the last one but thank you for the ones you did do they helped a lot getting into the game ❤️
